src/scenes/Partners/Detail/PartnerDetail.graphql

```diff
@@ -0,0 +1,20 @@
+query Partner($input: ID!) {
+  partner(id: $input) {
+    ...partnerDetails
+  }
+}
+
+fragment partnerDetails on Partner {
+  id
+  createdAt
+  organization {
+    canRead
+    canEdit
+    value {
+      name {
+        ...ss
+      }
+      avatarLetters
+    }
+  }
```

Comment:
I added the PartnerLookupItem fragment on CreatePartner but I don't think we can combine the rest.  The rest are all separate components/scenes that shouldn't share fragments.

One idea I had was to have a shared fragments file that we can store common fragments in, like all the secured stuff (securedString, securedNum, etc).  In this case we can but securedOrganization in there ```organization {
    canRead
    canEdit
    value {
      name {
        ...ss
      }
    }
  }```

Reply:
Yeah, that seems reasonable. We'd probably want to discuss such a centralized fragments folder as a group, but it seems like it could really de-clutter and also make typings a little more consistent.